src/components/CourseCard.tsx

```diff
@@ -14,54 +18,63 @@ export const CourseCard = ({
 }) => {
   const router = useRouter();
   const imageUrl = course.imageUrl ? course.imageUrl : 'banner_placeholder.png';
+  const percentage= course.totalVideos !== undefined ? Math.ceil(((course.totalVideosWatched ?? 0) / course?.totalVideos) * 100) : 0;
+
   return (
-    <div
-      className={`flex w-full cursor-pointer flex-col rounded-2xl bg-primary/5 transition-all duration-300 hover:-translate-y-2 hover:border-primary/20`}
-      onClick={onClick}
-    >
-      <div className="flex flex-col">
-        <img
-          alt={course.title}
-          className="size-full rounded-t-2xl bg-cover"
-          src={imageUrl}
-        />
+    <Card className="w-full max-w-sm overflow-hidden transition-all hover:shadow-lg cursor-pointer" onClick={onClick}>
+      <div className="relative aspect-video overflow-hidden">
+        <img 
+  alt={course.title} 
```

Comment:
Can you please fix the indentation?

Reply:
I’m really sorry, I didn’t realize the code wasn’t properly indented.
but i will fix indentation and then submit again